services/metadata/frontend/CookiesModal.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  import { cookiesAgreement, getCookie, setCookie } from "./cookies-service";
+
+  let modalOn = getCookie('cookiesAgreement') ? false : true;
+
+  const handleModal = (all?: boolean) => {
+    modalOn = !modalOn;
+    if(all) {
+      cookiesAgreement.set(true);
+      window.gtag.update();
+      setCookie('cookiesAgreement', 'true');
+    } else {
+      setCookie('cookiesAgreement', 'false');
+    }
+  };
+</script>
+
+{#if modalOn}
+  <div id="cookieConsent">
+    <div class="cookie-consent-modal">
+      <div class="modal-content-wrapper">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h3 class="modal-title">Cookies settings</h3>
+          </div>
+          <div class="modal-body">
+            We use cookies to personalize content and analyze access to our websites. Please choose whether you 
+            accept only cookies that are necessary for the functioning of the website or whether you accept also
+            analytical cookies. For more information, please refer to our
+            <a href="https://dasch.swiss/privacy-policy/" target=_blank>privacy policy</a>.
```

Comment:
Could you please also add links to the EULA: dasch.swiss/eula/ and Cookie Policy: dasch.swiss/cookie-policy?

Reply:
Done in c1d7e78.

Reply:
Actually should we call it `cookie` or `cookies policy`? To make it consistent I either change modal header where it's `cookies settings` now or link and description need to be changed.

Reply:
sorry, I wasn't very clear. We will probably have all three: EULA, privacy policy, and cookies policy. @danielasubotic is going to check what exactly is needed next week. Maybe we can merge privacy policy and cookies policy into one document.